Dockerfile

```diff
@@ -62,6 +62,10 @@ COPY --from=amd64 /src/open-saves/build/collector /collector
 
 CMD ["/collector"]
 
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.12 && \
+    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
+    chmod +x /bin/grpc_health_probe
+
 # Build the server image.
```

Comment:
The executable should probably be packaged in both the collector and the server.
For the collector, I would add this section at line 62 and line 78 in the original file, and keep the CMD as the last step.

Reply:
Done